.vscode/launch.json

```diff
@@ -14,7 +14,8 @@
             },
             "windows": {
                 "program": "ark.exe"
-            }
+            },
+            "sourceLanguages": ["rust"]
```

Comment:
I think this is a neutral-to-positive move for us.

Reply:
It looks like this enables these extra features, including formatters: https://github.com/vadimcn/codelldb/blob/b412cef3db6102591579d11c866ad60cae3fd65a/lang_support/rust.py#L10